deposit/src/main/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJob.java

```diff
@@ -560,12 +576,50 @@ private void addDescription(ContentObject obj) throws IOException {
 	}
 	
 	private void addPremisEvents(ContentObject obj) {
-		File premisFile = new File(getEventsDirectory(), obj.getPid().getUUID() + ".xml");
+		File premisFile = new File(getEventsDirectory(), obj.getPid().getUUID() + ".ttl");
 		if (!premisFile.exists()) {
 			return;
 		}
-		FilePremisLogger premisLogger = new FilePremisLogger(obj.getPid(), premisFile, repository);
+		PremisLogger premisLogger = getPremisLogger(obj.getPid());
 		List<PremisEventObject> events = premisLogger.getEvents();
+		// add premis events to fedora
 		obj.addPremisEvents(events);
 	}
+	
+	private void addIngestionEventForContainer(ContentContainerObject obj, Resource parentResc) throws IOException {
+		NodeIterator childIt = getChildIterator(parentResc);
+		int numChildren = 0;
+		PID childPid = null;
+		while (childIt.hasNext()) {
+			Resource resc = (Resource) childIt.next();
+			// we need the pid only if there is exactly one child
+			if (numChildren < 1) {
+				childPid = PIDs.get(resc.getURI());
+			}
+			numChildren++;
+		}
+		// don't add event unless at least one child is ingested
+		if (numChildren > 0) {
+			PremisLogger premisLogger = getPremisLogger(obj.getPid());
+			PremisEventBuilder builder = premisLogger.buildEvent(Premis.Ingestion);
```

Comment:
Sorry to add one more thing, but I was reminded that these events need to record what agents were responsible for the event.  This should be a copypasta job from IngestDepositRecordJob at lines 84 and 85:

```
.addSoftwareAgent(SoftwareAgent.depositService.getFullname())
.addAuthorizingAgent(DepositField.depositorName.name())
```

That same is needed in `addIngestionEventForChild`